comparison.md

```diff
@@ -7,20 +7,33 @@ url: /comparison
 
 ## Overview
 
-Truth is a fluent assertion framework, which has a lot of similarities to
-[AssertJ], which was forked from [FEST]. Truth is significantly different from
-[Hamcrest], which uses [Polish prefix notation] to compose assertions.
+As a [fluent] assertion library, Truth is similar to [AssertJ], which was forked
+from [FEST]. Truth is significantly different from [Hamcrest], which uses
+[Polish prefix notation] to compose assertions.
 
 
-### Why create Truth when AssertJ already exists?
+## Why create Truth when AssertJ already exists?
 
 In other words: Maybe Truth is better for some use cases, but why not just
 improve AssertJ instead? The reason is historical: AssertJ didn't exist when we
 started Truth. By the time it was created, we'd begun using Truth widely
 internally, and we'd added some features that would be difficult to retrofit
 onto AssertJ.
 
-## Feature Comparison vs. AssertJ
+## Comparison vs. Hamcrest
+
+Because Truth and Hamcrest differ so significantly, I'll cover only the main
```

Comment:
I think people would be confused by the usage of "I'll" here, as it suggests to me at least that they ought to know who wrote the doc, which I'm sure was not intended. :)

How about replacing it with "we'll"?

Or, even better, how about replacing the entire sentence with something like the following?

> Because Truth and Hamcrest differ so significantly, only the main differences are listed here: